scylla-cql/src/value.rs

```diff
@@ -1180,6 +1181,11 @@ impl std::fmt::Display for CqlValue {
                     .fmt(f)?;
                 f.write_str("}")?;
             }
+            CqlValue::Vector(v) => {
+                f.write_str("[")?;
+                v.iter().format(",").fmt(f)?;
+                f.write_str("]")?;
+            }
```

Comment:
Nit: you could extend the existing List case instead of introducing a separate one, they have exactly the same syntax:

```rust
CqlValue::List(v) | CqlValue::Vector(v) => {
    // ...
}
```